ledger/db/src/tx_out_store.rs

```diff
@@ -796,7 +796,7 @@ pub mod tx_out_store_tests {
     use std::path::Path;
     use tempdir::TempDir;
 
-    fn get_env() -> Environment {
+    pub fn get_env() -> Environment {
```

Comment:
If we're making this a public function, maybe we should add a doc comment? Or is it not necessary here?

Reply:
I'll add a comment, but in general I don't think we made it a priority to document test code thoroughly.